controller/examples/examples.go

```diff
@@ -209,6 +210,13 @@ func (e *generator) updateApp() {
 	e.client.UpdateApp(app)
 }
 
+func (e *generator) getAppLog() {
+	res, err := e.client.GetAppLogWithWait(e.resourceIds["app"], 0, false)
+	if err == nil {
+		io.Copy(ioutil.Discard, res)
+	}
+}
```

Comment:
Thanks. I also wasn't 100% sure what to do with these "examples" or how they're used. The rest of it looks ok though?